### DISCUSSION TITLE
EmuELEC.Guide/more/88.change_system

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#714] [Announcements]
# EmuELEC.Guide/more/88.change_system

EmuELEC的玩法介绍

https://ryukarin.github.io/EmuELEC.Guide/more/88.change_system.html

**Comment by dashu1995** [comment]
我在使用EmuELEC系统时，遇到了这篇博文，如获至宝片。当我下载博主的解包打包脚本工具system.sh，发现链接已经失效。我想求助博主可否把“解包打包脚本工具system.sh文件”发我邮箱156441722@qq.com。或者在方便时更新一下工具链接。感谢！

**Reply by dashu1995** [reply]
看了下博文的发布时间，已经有1年多了。希望有天博主看到我的留意后。如果还有解包打包脚本工具system.sh脚本，请帮忙发至我的邮箱156441722@qq.com。谢谢
